lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -138,9 +138,8 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
 
   /// The current list of windows,
```

Comment:
The comma here looks unintentional, consider fixing it while you're here :)

```suggestion
  /// The current list of windows.
```